ui/apps/platform/src/Containers/ComplianceEnhanced/Status/Dashboard/ScanResultsOverviewTable.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useCallback } from 'react';
+import {
+    ActionsColumn,
+    IAction,
+    TableComposable,
+    Tbody,
+    Td,
+    Th,
+    Thead,
+    Tr,
+} from '@patternfly/react-table';
+import { format } from 'date-fns';
+
+import useRestQuery from 'hooks/useRestQuery';
+import useURLPagination from 'hooks/useURLPagination';
+import useURLSort from 'hooks/useURLSort';
+import useURLSearch from 'hooks/useURLSearch';
+import { complianceResultsOverview } from 'services/ComplianceEnhancedService';
+import { SortOption } from 'types/table';
+
+import { Bullseye, Spinner } from '@patternfly/react-core';
+import ScanResultsToolbar from './ScanResultsToolbar';
+
+const sortFields = ['Scan Name', 'Failing Controls', 'Last Scanned'];
+const defaultSortOption = { field: 'Scan Name', direction: 'asc' } as SortOption;
+
+function ScanResultsOverviewTable() {
+    const { page, perPage, setPage, setPerPage } = useURLPagination(10);
+    const { sortOption, getSortParams } = useURLSort({
+        sortFields,
+        defaultSortOption,
+    });
+    const { searchFilter, setSearchFilter } = useURLSearch();
+
+    const listQuery = useCallback(
+        () => complianceResultsOverview(searchFilter, sortOption, page - 1, perPage),
+        [searchFilter, sortOption, page, perPage]
+    );
+    const { data: scanResultsOverviewData, loading: isLoading } = useRestQuery(listQuery);
+
+    const defaultActions = (): IAction[] => [
+        {
+            title: 'Edit schedule',
+            // TODO: link to edit schedule page
+        },
+    ];
+
+    const displayOnlyItemOrItemCount = (items: string[], multipleItemLabel: string): string => {
+        if (items.length > 1) {
+            return `${items.length} ${multipleItemLabel}`;
+        }
+        return items[0];
+    };
+
+    const renderTableContent = () => {
+        if (isLoading) {
+            return (
+                <Tr>
+                    <Td colSpan={8}>
+                        <Bullseye>
+                            <Spinner isSVG />
+                        </Bullseye>
+                    </Td>
+                </Tr>
+            );
+        }
+
+        return scanResultsOverviewData?.map(({ scanStats, clusterId, profileName }) => (
+            // TODO: verify scanName unique
+            <Tr key={scanStats.scanName}>
```

Comment:
That is the intent.  scanName would be the name of the configuration created within the app.  For the short term it would return the latest set of results.  Gives us a way to logically marry results with the configuration that created them.

Reply:
@dashrews78 Can the scan name be modified? We're planning to add an ID to the URL for report viewing. Ideally, we'd prefer a unique integer in the request, especially if users can change the scan name. 

[Here's a PR](https://github.com/stackrox/stackrox/pull/7756) to get an idea of what I'm referring to

Reply:
@bradr5 The way we wire it up with the compliance operator objects through sensor is that name is unique.  We could switch to a UUID if that makes the UI life simpler.  (I actually had a UUID there at one point.). I would just have to take care on how to map the compliance operator objects we generate with what the interface and user experience needs.  Certainly doable to store and keep that abstracted away from the UI, just not as simple.  The fact that the compliance operator objects use name fields for the CRDs is what drove down this path.  Is there really a use case to rename a scan config?

Regardless of all those words, scanName would still need to be unique to eliminate any confusion.

Reply:
I _believe_ a UUID would be more in line with how we typically handle urls but not required. Let me get back to on that.